assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -190,6 +185,15 @@ function BannerNotification( {
 		if ( isDismissible && dismissOnCTAClick ) {
 			dismissNotification();
 		}
+
+		if (
```

Comment:
This block is moved below the `onCTAClick` call, with the addition of the ` e.defaultPrevented` condition, to avoid interfering with `onCTAClick`.